### PR TITLE
Enable sendrawtransaction to send IS

### PR DESCRIPTION
**Pull request by thelazier**

### CODE REVIEW

File: src/rpcrawtransaction.cpp [hunk 1]
```diff
@@ -865,6 +866,10 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
     } else if (fHaveChain) {
         throw JSONRPCError(RPC_TRANSACTION_ALREADY_IN_CHAIN, "transaction already in block chain");
     }
+    if (IsInstantSendTxValid(tx)) {
```
Comment (UdjinM6) [inline src/rpcrawtransaction.cpp]: 1. I don't think this should be the default behavior (at least for now), I think we should either add another optional param to `sendrawtransaction` smth like 
```
...
"sendrawtransaction \"hexstring\" ( allowhighfees instantsend )\n"
...
"3. instantsend      (boolean, optional, default=false) Use InstantSend to send this transaction\n"
...
bool fInstantSend = false;
if (params.size() > 2)
    fInstantSend = params[2].get_bool();
```
plus some param description in help.
(or create new rpc `instantsendrawtransaction` like `instantsendtoaddress` we already have, but that's probably an overkill here)

2. when (1) is done `if (IsInstantSendTxValid(tx)) { ...` should rather become
```
if (fInstantSend) {
    if (!IsInstantSendTxValid(tx)) {
        throw JSONRPCError(RPC_TRANSACTION_ERROR, "Not a valid InstantSend transaction");
    }
    mapLockRequestAccepted.insert(make_pair(hashTx, tx));
    CreateTxLockCandidate(tx);
}
```
or smth like that

File: src/rpcrawtransaction.cpp [hunk 2]
```diff
@@ -810,14 +811,15 @@ UniValue signrawtransaction(const UniValue& params, bool fHelp)
...
+    if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
-            "sendrawtransaction \"hexstring\" ( allowhighfees )\n"
+            "sendrawtransaction \"hexstring\" ( allowhighfees instantsend )\n"
             "\nSubmits raw transaction (serialized, hex-encoded) to local node and network.\n"
             "\nAlso see createrawtransaction and signrawtransaction calls.\n"
             "\nArguments:\n"
             "1. \"hexstring\"    (string, required) The hex string of the raw transaction)\n"
-            "2. allowhighfees    (boolean, optional, default=false) Allow high fees\n"
+            "2. allowhighfees  (boolean, optional, default=false) Allow high fees\n"
```
Comment (UdjinM6) [inline src/rpcrawtransaction.cpp]: messed up white spaces (next line too), otherwise looks good 👍

Comment (thelazier) [inline src/rpcrawtransaction.cpp]: It's currently messed up and I thought I fixed them (existing line and my line) The 1st has `\"` twice which will be less 2 space on real output. 
FYI, current help message;
```$ dash-cli help sendrawtransaction
sendrawtransaction "hexstring" ( allowhighfees )

Submits raw transaction (serialized, hex-encoded) to local node and network.

Also see createrawtransaction and signrawtransaction calls.

Arguments:
1. "hexstring"    (string, required) The hex string of the raw transaction)
2. allowhighfees    (boolean, optional, default=false) Allow high fees

Result:
"hex"             (string) The transaction hash in hex

Examples:

Create a transaction
> dash-cli createrawtransaction "[{\"txid\" : \"mytxid\",\"vout\":0}]" "{\"myaddress\":0.01}"
Sign the transaction, and get back the hex
> dash-cli signrawtransaction "myhex"

Send the transaction (signed hex)
> dash-cli sendrawtransaction "signedhex"

As a json rpc call
> curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "sendrawtransaction", "params": ["signedhex"] }' -H 'content-type: text/plain;' http://127.0.0.1:9998/
```

Comment (UdjinM6) [inline src/rpcrawtransaction.cpp]: hah! I didn't notice! 👍

**Pull request merged**